test/dummy-jira.js

```diff
@@ -58,10 +59,30 @@ class DummyJira {
       SubTask13: issueGenerator('SubTask13', 'Sub-task', 'green', 'MT1'),
       SubTask14: issueGenerator('SubTask14', 'Sub-task', 'green', 'Bug1'),
       SubTask15: issueGenerator('SubTask15', 'Sub-task', 'green'),
+
+      //test to see if the code cares about broken children
+      //SubTask16: issueGenerator('SubTask16', 'Sub-task', 'red','Story1'),
```

Comment:
I'm confused as to the value of these comments you left here. I know there other comments here that we could do without as well, but what purpose do these serve?